pkg/ec2/test_env

```diff
@@ -1,7 +1,8 @@
-export AMI=TODO
+export AMI=ami-0ca285d4c2cda3300
```

Comment:
This file is meant to serve as an example, so let's drop any changes to this file from the PR.